src/pages/Homepage/Homepage.css

```diff
@@ -2,36 +2,28 @@
   background-color: var(--primary-color);
   border: none;
   border-radius: 5px;
-  color: white;
-  color: var(--background-color);
+  color: var(--text-color);
   cursor: pointer;
   font-size: 2rem;
   height: 5rem;
   width: 100%;
   margin-bottom: 4rem;
   padding: 3px 20px;
```

Comment:
You can change the 20px to rem if you want.

Reply:
+1

Reply:
done